sbin/common-functions.sh

```diff
@@ -101,6 +101,27 @@ checkingAndDownloadingFreeType()
     else
       # shellcheck disable=SC2154
       echo "${good}Successfully configured OpenJDK with the FreeType library (libfreetype)!"
+
+     if [[ ${OS_KERNEL_NAME} == "darwin" ]] ; then
+        TARGET_DYNAMIC_LIB_DIR="${WORKING_DIR}"/"${OPENJDK_REPO_NAME}"/installedfreetype/lib/
+        TARGET_DYNAMIC_LIB="${TARGET_DYNAMIC_LIB_DIR}"/libfreetype.6.dylib
+        echo ""
+        echo "Listing the contents of ${TARGET_DYNAMIC_LIB_DIR} to see if the dynamic library 'libfreetype.6.dylib' has been created..."
+        ls "${TARGET_DYNAMIC_LIB_DIR}"
+
+        echo ""
+        echo "Releasing the runpath dependency of the dynamic library ${TARGET_DYNAMIC_LIB}"
+        set -x
+        install_name_tool -id @rpath/libfreetype.6.dylib "${TARGET_DYNAMIC_LIB}"
```

Comment:
@mbvreddy After all the conversations on issue  #202, here's our current solution, I have reverted the change to include @rpath in parameter.

You mentioned it worked for you, could you please share your command history with us, so I can apply them here.

I have seen @smlambert's workaround, unsure if I can include them in the build.